int32 tag_id
string marker_topic
geometry_msgs/PoseStamped base_pose_goal
